tests/test_tableformat.py

```diff
@@ -0,0 +1,85 @@
+from src.tableformat import TextTableFormatter, CSVTableFormatter
+import pytest
+
+
+@pytest.fixture
+def data():
+    headers = ("Name", "Shares", "Price", "Change")
+    rows = [
+        ("AA", 100, 9.22, -22.98),
+        ("IBM", 50, 106.28, 15.18),
+        ("CAT", 150, 35.46, -47.98),
+        ("MSFT", 200, 20.89, -30.34),
+        ("GE", 95, 13.48, -26.89),
+        ("MSFT", 50, 20.89, -44.21),
+        ("IBM", 100, 106.28, 35.84),
+    ]
+    return headers, rows
+
+
+def test_text_table_formatter(data):
+    """
+    Tests if the TextTableFormatter returns the correctly formatted table.
+    """
+    headers, rows = data
+    expected = [
+        "      Name     Shares      Price     Change",
+        "---------- ---------- ---------- ----------",
+        "        AA        100       9.22     -22.98",
+        "       IBM         50     106.28      15.18",
+        "       CAT        150      35.46     -47.98",
+        "      MSFT        200      20.89     -30.34",
+        "        GE         95      13.48     -26.89",
+        "      MSFT         50      20.89     -44.21",
+        "       IBM        100     106.28      35.84",
+    ]
+    txt_formatter = TextTableFormatter()
+    print(headers, rows)
```

Comment:
I guess it was only for debugging purpose, you should remove it now

Reply:
Removed.